src/manifolds/MultinomialDoublyStochastic.jl

```diff
@@ -0,0 +1,286 @@
+@doc raw"""
+    MultinomialDoublyStochastic{n} <: AbstractEmbeddedManifold{ℝ, DefaultEmbeddingType}
+
+The set of doubly stochastic multinomial matrices consists of all symmetricThe multinomial manifold consists of `m` column vectors, where each column is of length
+`n` and unit norm, i.e.
+
+````math
+\begin{aligned}
+\mathcal{DP}(n) \coloneqq \bigl\{p ∈ ℝ^{n×n}\ \big|\ &p_{i,j} > 0 \text{ for all } i=1,…,n, j=1,…,m,\\
+& p\mathbf{1}_n = p^{\mathrm{T}}\mathbf{1}_n = \mathbf{1}_n
+\bigr\},
+\end{aligned}
+````
+
+where $\mathbf{1}_n$ is the vector of length $n$ containing ones.
+
+The tangent space can be written as
+
+````math
+T_p\mathcal{DP}(n) \coloneqq \bigl\{
+X ∈ ℝ^{n×n}\ \big|\ X = X^{\mathrm{T}} \text{ and }
+X\mathbf{1}_n = X^{\mathrm{T}}\mathbf{1}_n = \mathbf{0}_n
+\bigr\},
+````
+
+where $\mathbf{0}_n$ is the vector of length $n$ containing zeros.
+
+More details can be found in Section III[^DouikHassibi2019].
+
+# Constructor
+
+    MultinomialDoubleStochasticMatrices(n)
+
+Generate the manifold of matrices $\mathbb R^{n×n}$ that are doubly stochastic and symmetric.
+
+[^DouikHassibi2019]:
+    > A. Douik, B. Hassibi:
+    > Manifold Optimization Over the Set of Doubly Stochastic Matrices: A Second-Order Geometry,
+    > IEEE Transactions on Signal Processing 67(22), pp. 5761–5774, 2019.
+    > doi: [10.1109/tsp.2019.2946024](http://doi.org/10.1109/tsp.2019.2946024),
+    > arXiv: [1802.02628](https://arxiv.org/abs/1802.02628).
+"""
+struct MultinomialDoubleStochasticMatrices{N} <:
+       AbstractEmbeddedManifold{ℝ,DefaultEmbeddingType} where {N} end
+
+function MultinomialDoubleStochasticMatrices(n::Int)
+    return MultinomialDoubleStochasticMatrices{n}()
+end
+
+@doc raw"""
+    check_manifold_point(M::MultinomialDoubleStochasticMatrices, p)
+
+Checks whether `p` is a valid point on the [`MultinomialDoubleStochasticMatrices`](@ref)`(m,n)` `M`,
+i.e. is a  matrix with positive entries whose rows and columns sum to one.
+"""
+check_manifold_point(::MultinomialDoubleStochasticMatrices, ::Any)
+function check_manifold_point(
+    M::MultinomialDoubleStochasticMatrices{n},
+    p;
+    kwargs...,
+) where {n}
+    mpv =
+        invoke(check_manifold_point, Tuple{supertype(typeof(M)),typeof(p)}, M, p; kwargs...)
+    mpv === nothing || return mpv
+    c = sum(p, dims = 1) # due to symmetry we only have to check the cols
```

Comment:
What symmetry do you mean here?

Reply:
see first amswer, I'll revise that comment.